ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/MessageMouseArea.qml

```diff
@@ -4,11 +4,15 @@ import "../../../../../imports"
 
 MouseArea {
     cursorShape: chatText.hoveredLink ? Qt.PointingHandCursor : undefined
-    acceptedButtons: Qt.RightButton
+    acceptedButtons: Qt.RightButton | Qt.LeftButton
     z: 50
     onClicked: {
         if(mouse.button & Qt.RightButton) {
-            clickMessage(false, isSticker)
+            clickMessage(false, isSticker, false);
+            return;
+        } else if (mouse.button & Qt.LeftButton) {
```

Comment:
Not a big deal, but it's cleaner this way. Since you already return, the `else` is not needed
```suggestion
        }
        if (mouse.button & Qt.LeftButton) {
```